torch/fx/experimental/proxy_tensor.py

```diff
@@ -82,29 +92,35 @@ def unwrap_proxy(e):
         args[0].proxy = proxy_out
         proxy_out.node.meta['tensor_meta'] = _extract_tensor_metadata(args[0])
 
-    with no_dispatch():
-        real_out = func_overload(*args, **kwargs)
-
+    real_out = func_overload(*pytree.tree_map(unwrap_elem, args), **pytree.tree_map(unwrap_elem, kwargs))
+    # Seems like func isn't marked as inplace_view for some reason?
+    if torch.Tag.inplace_view in func_overload.tags:  # type: ignore[attr-defined]
+        with no_dispatch():
+            func_overload(*args, **kwargs)
+    pytree.tree_map(lambda x: check_metadata_consistency(x, ProxyTensor), (real_out, args, kwargs))
```

Comment:
nit: The error message for this might be unintelligible for non-pytorch developers (because it was written just for pytorch developers). If any users will see this then you might want to make it better

Reply:
yeah, it wouldn't be the worst if I just removed this.... I think.